apis/Google.LongRunning/Google.LongRunning.Snippets/OperationsClientSnippets.g.cs

```diff
@@ -53,9 +53,9 @@ public class GeneratedOperationsClientSnippets
             // Create client
             OperationsClient operationsClient = OperationsClient.Create();
             // Initialize request argument(s)
-            string formattedName = OperationsClient.FormatOperationPathName("[OPERATION_PATH]");
+            string name = "";
             // Make the request
-            Operation response = await operationsClient.GetOperationAsync(formattedName);
+            Operation response = await operationsClient.GetOperationAsync(name);
```

Comment:
Given the license issue, it's entirely possible that something odd has happened in terms of autogen. Will look carefully when I get the chance. (I'm not on the right computer to check right now.)